substrate/frame/src/lib.rs

```diff
@@ -60,6 +60,7 @@
 //! }
 //! ```
 //!
+//! test
```

Comment:
Can you maybe try that the `bump: none` works now?

Reply:
seems to work